package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "serve": "node scripts/start-web-server.js --port 8080",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
+    "lint": "vue-cli-service lint 'src/**' 'cypress/**'",
```

Comment:
Čo sa tu stane ?

Reply:
I just specified the correct paths to linter - so it'll catch all files in the `src`  and the `cypress` and it'll lint it (check the syntax against the pre-set rules).